misk-api/src/main/kotlin/misk/web/HttpCall.kt

```diff
@@ -35,6 +35,7 @@ interface HttpCall {
   var requestHeaders: Headers
 
   /** Cookies derived from request's "Cookie" header, if any */
+  //TODO: Remove cookies
```

Comment:
Could more reasoning for why be added here?

Reply:
@rainecp changed this to use okhttp3 cookie instead.